aries-site/src/examples/templates/export-data/ExportDataContainer.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState, useContext } from 'react';
+import {
+  AnnounceContext,
+  Box,
+  Button,
+  Form,
+  FormField,
+  RadioButtonGroup,
+  Paragraph,
+} from 'grommet';
+import { ModalBody, ModalDialog, ModalFooter } from 'aries-core';
+
+import PropTypes from 'prop-types';
+
+export const ExportDataContainer = ({
+  onSubmit: onSubmitProp,
+  setShowModal,
+  title,
+}) => {
+  // announce when the layer opens
+  const announce = useContext(AnnounceContext);
+  useEffect(() => {
+    announce(`${title} modal opened.`, 'assertive');
+  }, [announce, title]);
+
+  const [value, setValue] = useState('');
+  const onClose = () => {
+    setShowModal(false);
+    setValue('');
+  };
+  const onChange = nextValue => {
+    setValue(nextValue);
+  };
+
+  return (
+    <ModalDialog
+      title={title}
+      onEsc={() => {
+        announce(`${title} modal cancelled and closed.`, 'assertive');
+        onClose();
+      }}
+    >
+      <Form
+        value={value}
+        onChange={onChange}
+        validate="blur"
+        // eslint-disable-next-line no-unused-vars
+        onSubmit={({ value: formValue, touched }) => {
+          if (onSubmitProp) onSubmitProp(formValue, touched);
+          onClose();
+        }}
+      >
+        <Box gap="medium">
+          <ModalBody gap="small">
+            <Paragraph margin="none">Select file type.</Paragraph>
```

Comment:
It feels like this Paragraph should sit right under the modal heading as opposed to in the form. Alternately, we could just use the `label` prop on the formfield and say `label="File type"`?

This is more of a design reaction.

Reply:
Landed on using FormField's `label=File type`